runtime-manager/Cargo.toml

```diff
@@ -131,6 +132,7 @@ wasmi = { git = "https://github.com/veracruz-project/wasmi.git", branch = "verac
 
 [build-dependencies]
 uuid = { version = "0.7", features = ["v4"] }
+libloading = "=0.5.2"
```

Comment:
nit: can you move `libloading` above `uuid` to keep alphabetical order of imports after the recent formatting?